src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -1678,20 +1711,20 @@ class CosmasGoogleCloudServiceTest {
     }
 
     private fun getVersionsList(fileId: String = FILE_ID, projectId: String = PROJECT_ID,
-                                startGeneration: Long = -1L, isFreePlan: Boolean = true): List<Long> {
+                                startGeneration: Long = -1L, isFreePlan: Boolean = true, cosmas: CosmasGoogleCloudService = this.service): List<Long> {
```

Comment:
This line looks too long, i think(so 1723)

Reply:
fixed